### ISSUE TITLE
Inference time too long on ade20k validation set (HRNet + OCR)

### ISSUE THREAD

**Issue by lubovbyc** [HRNet/HRNet-Semantic-Segmentation#204]
![image](https://user-images.githubusercontent.com/5144935/101243531-00727a80-373c-11eb-8b22-b17b56f28f05.png)

dataset: ade20k validation set
device: single gpu (V100)
time: ~5h

Multi-scale inference seems too slow on this dataset. Is it normal?

**Comment by PkuRainBow** [comment]
@lubovbyc I guess this is normal as this implementation does not support multi-GPU testing. Please try the openseg.pytorch for multiscale evaluation with multi-GPUs.
